### PR TITLE
YarynaCherniak_ PublicGroupsUtilities and TriggerHandler

### PR DESCRIPTION
**Pull request by YarynaCherniak**

### CODE REVIEW

File: VotingProject/force-app/main/default/classes/VC_MembersUtilities_Queueble.cls [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    }
+    /*
+    @description: adds configurators to public group
+    @param  context
+    */
+    public void execute(QueueableContext context) {
+        List<GroupMember> configurators = new List<GroupMember>();
+        for (Group newGroup: newGroups){
+            if (newGroup?.name?.contains('Configurators')){
+                configurators.add(new GroupMember(GroupId=newGroup.Id, UserOrGroupId=UserInfo.GetUserId()));
```
Comment (Devillkiller) [inline VotingProject/force-app/main/default/classes/VC_MembersUtilities_Queueble.cls:25]: Move User Id getter before the loop body and store returned value in variable so System resources used wisely.

File: VotingProject/force-app/main/default/classes/VC_PublicGroupsUtilities.cls [hunk 2]
```diff
@@ -0,0 +1,62 @@
+/*
+    Apex Class Name:  VC_PublicGroupsUtilities
+                      (Namespace "VC" stands for Voting Campaign)
+    @description: This class is designed for creating/deleting public groups for each
+                  voting campaign user role after campaign creation/deletion
+*/
+public with sharing class VC_PublicGroupsUtilities {
```
Comment (Devillkiller) [inline VotingProject/force-app/main/default/classes/VC_PublicGroupsUtilities.cls:7]: No need to use "static" methods/properties here. Please update.

File: VotingProject/force-app/main/default/classes/VC_PublicGroupsUtilities.cls [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+        for(Voting_Campaign__c oldCampaign :oldCampaigns){
+            oldCampaignsNames.add('%' + oldCampaign.Name + '%');
+        }
+        // Filtering groups according to the deleted campaign names
+        List<Group> groupsToBeDeleted = [SELECT ID FROM Group WHERE Name LIKE :oldCampaignsNames];
+        
+        try {
+            delete groupsToBeDeleted;
+        } 
+        catch(QueryException queryE) {
```
Comment (Devillkiller) [inline VotingProject/force-app/main/default/classes/VC_PublicGroupsUtilities.cls:55]: Why not to handle using Exception Utils?

**Review by Devillkiller — CHANGES_REQUESTED** [review]

**Comment by YarynaCherniak** [comment]
> No need to use "static" methods/properties here. Please update. 

	"resource": "/force-app/main/default/classes/VotingCampaignTriggerHandler.cls",
	"message": "Non static method cannot be referenced from a static context: void VC_PublicGroupsUtilities.createPublicGroups(List<Voting_Campaign__c>) (27:34)",
VC_PublicGroupsUtilities.deletePublicGroups(List<Voting_Campaign__c>) (32:34)

**Pull request merged**